tools/ebpf/Dockerfiles/Dockerfile-process-agent-dev

```
@@ -0,0 +1,10 @@
+FROM $base_image
```

I think you can use the docker build "--build-arg" feature here, like this you don't need to create a temporary dockerfile.

```suggestion
ARG AGENT_BASE=datadog/agent:latest
FROM $AGENT_BASE
```

and then `docker build --build-arg AGENT_BASE=<base_image>`

---

Much cleaner, done